pkg/controller/controller.go

```diff
@@ -92,17 +92,18 @@ type ErrorDetails struct {
 
 const iapHeaderName = "x-goog-authenticated-user-email"
 
-func New(h *renderer.Renderer, p *justification.Processor, allowlist []string) *Controller {
-	categories := make(map[string]struct{})
-	for v := range p.Validators() {
-		categories[v] = struct{}{}
+func New(ctx context.Context, h *renderer.Renderer, p *justification.Processor, allowlist []string) (*Controller, error) {
+	categories, err := catagoriesDisplayData(ctx, p.Validators())
```

Comment:
Sanity check whether the plugin is ready to serve data at initialization. Could you run it locally to ensure it is working?

Reply:
Confirmed offline that it would work :D Resolving this thread now